guacamole/src/main/webapp/translations/fr.json

```diff
@@ -98,7 +111,8 @@
         "HELP_MOUSE_MODE"          : "Détermine comment la souris distante se comporte selon les événements.", 
         "HELP_MOUSE_MODE_ABSOLUTE" : "Appuyer pour cliquer. Le clique s'effectue à l'endroit de l'appui.",
         "HELP_MOUSE_MODE_RELATIVE" : "Glisser pour déplacer le pointeur de la souris et appuyer opur cliquer. Le clique s'effectue à l'endroit du pointeur.", 
-
+        "HELP_SHARE_LINK"          : "La connexion actuelle est en cours de partage et est accessible à toute personne possédant les caractéristiques suivantes: {LINKS, plusieurs, un{link} autres{links}}:",
```

Comment:
In keeping with style of the rest of the files, there should be a blank line after this one.